content/workshops/rhel_8/exercise2.0.adoc

```diff
@@ -0,0 +1,126 @@
+---
+title: "Exercise 2.0 - Installing Ansible Tower"
+workshops: rhel_8
+workshop_weight: 20
+layout: lab
+---
+
+:license_url: http://ansible-workshop-bos.redhatgov.io/ansible-license.json
+
+:icons: font
+:imagesdir: /workshops/rhel_8/images
+
+
+= Exercise 2.0 - Installing Ansible Tower
+
+== Exercise Description
+
+In this exercise, we are going to get Ansible Tower installed on your control node.
```

Comment:
Line 18 (add) Red Hat Ansible helps teams manage complex, multitier deployments by adding control, knowledge, and delegation to Ansible-powered environments. Red Hat Ansible Tower helps you scale IT automation, manage complex deployments, and speed productivity. In this exercise, we will install Ansible Tower on your control node.

Reply:
Removed this section